### PR TITLE
Amended too long lines

### PR DESCRIPTION
**Pull request by jremes-foss**
Hi.

Looks like there was a lot of too long lines in ```sympy/calculus/util.py```.

These have been now amended. 

```./bin/test quality``` ran locally - all green.

Thanks.

### CODE REVIEW

File: sympy/calculus/util.py [hunk 1]
```diff
@@ -60,20 +60,20 @@ def continuous_domain(f, symbol, domain):
     try:
         sings = S.EmptySet
         if f.has(Abs):
-            sings = solveset(1/f, symbol, domain)
+            sings = solveset(1 / f, symbol, domain)
```
Comment (smichr) [inline sympy/calculus/util.py]: we leave spaces around `+` and `-` but not `/` and `*`. Can you remove this changes?

Comment (Newman101) [inline sympy/calculus/util.py]: Sure. I'll fix these ASAP.

Comment (Newman101) [inline sympy/calculus/util.py]: That should be sorted now.

**Comment by smichr** [comment]
It's in.

**Pull request merged**